src/test/testarb.annotated

```diff
@@ -0,0 +1,92 @@
+{
+    "@@locale": "en",
+    "appName": "Demo app",
+    "pageLog{inUsername": "Your username",
+     ^^^^^^^^^^^^^^^^^^[Error]
```

Comment:
Would there be value in including the diagnostic messages? Re-generating them if the text changes would be fairly cheap, but it would give some additional context (and make it easier for reviewers to know the generated messages look sensible).

Reply:
I was unsure, as they add noise, but it's not like people are reading through the whole golden test file. You are right!